www/components/Nav/index.tsx

```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
               </div>
               <div className="hidden lg:flex items-center sm:space-x-3">
                 <a href="https://github.com/supabase/supabase" target="_blank">
-                  <Button type="default" icon={<IconStar />}>
+                  <Button
+                    className="sbui-btn sbui-btn-text sbui-btn--tiny sbui-btn--text-align-center"
+                    icon={<IconStar />}
+                  >
                     Star us on GitHub
                   </Button>
                 </a>
+                <a href="https://app.supabase.io/" target="_blank">
+                  <Button type="default">Log in</Button>
```

Comment:
According to https://github.com/supabase/supabase/issues/4739#issuecomment-1006160419 should be `Sign in` to mirror the Supabase Studio label. 👍🏻

Reply:
Sure, I can change the nomenclature to `Sign in`  so that it matches with Studio Label.